src/detect-filesha1.c

```diff
@@ -37,7 +37,7 @@
 static int DetectFileSha1SetupNoSupport (DetectEngineCtx *a, Signature *b, const char *c)
 {
     SCLogError(SC_ERR_NO_SHA1_SUPPORT, "no SHA-1 calculation support built in, needed for filesha1 keyword");
-    return -1;
+    FAIL;
```

Comment:
The FAIL/PASS macro's should be limited to the tests themselves. Here they made it into non-test code.